### ISSUE TITLE
Parse error for string concatenation in object literal

### ISSUE THREAD

**Issue by atuttle** [cflint/CFLint#101]
Valid code in a CFM template:

	x = {
		name: rc.ticket.getShortId()
		,action: 'support:tickets/view/id/' & rc.ticket.getShortId()
	};

CLI lint result:

```
$ cflint -file view.cfm
line 8:38 no viable alternative at input '[{name:'Support Tickets',action:'support:tickets/home'},{name:rc.ticket.getShortId(),action:'support:tickets/view/id/'&'
<issues>
  <issue severity="ERROR" id="PARSE_ERROR" message="PARSE_ERROR" category="CFLint" abbrev="PE">
    <location file="/Users/adam/dev/{snip}/view.cfm" fileName="view.cfm" column="38" line="8" message="Unable to parse" variable="">
      <Expression><![CDATA[[@29,163:163='&',<54>,8:38]]]></Expression>
    </location>
  </issue>
</issues>
```

**Comment by ryaneberly** [comment]
Fixed in cfparser/cfparser master.

**Comment by ryaneberly** [comment]
fixed in 5.1-snapshot
